### PR TITLE
Add inCollection for ALMA NZ DE-655 #1677

### PR DESCRIPTION
**Pull request by TobiasNx**
I created a first version for adding a `inCollection`-object for ALMA NZ (See: #1677).
We can change the condition for this if necessary later.

### CODE REVIEW

File: src/main/resources/alma/fix/relatedRessourcesAndLinks.fix [hunk 1]
```diff
@@ -300,6 +300,13 @@ end
 
 set_array("inCollection[]")
 
+# hbz NZ
+
+if any_equal("003", "DE-605")
```
Comment (blackwinter) [inline src/main/resources/alma/fix/relatedRessourcesAndLinks.fix:305]: > We can change the condition for this if necessary later.

Do you mean this condition? It checks for `DE-605`, but assigns `DE-655`.

File: src/test/resources/alma-fix/990001412590206441.json [hunk 2]
```diff
@@ -68,6 +68,10 @@
     "id" : "http://digitale-objekte.hbz-nrw.de/storage2/2019/04/06/file_4/8295425.pdf"
   } ],
   "inCollection" : [ {
+    "id" : "http://lobid.org/organisations/DE-655",
```
Comment (blackwinter) [inline src/test/resources/alma-fix/990001412590206441.json:71]: The trailing `#!` is missing, right?

File: src/main/resources/alma/fix/relatedRessourcesAndLinks.fix (hunk 1 above)
Comment (TobiasNx) [reply src/main/resources/alma/fix/relatedRessourcesAndLinks.fix:305]: Yes it checks for DE-605 but DE-655 is the ISIL for ALMA-NZ: http://lobid.org/organisations/DE-655

And I used this as URI for the ALMA-NZ.

Comment (blackwinter) [reply src/main/resources/alma/fix/relatedRessourcesAndLinks.fix:305]: As long as we're clear that this is definitely not the final condition. `003=DE-605` includes local (IZ) records.

File: src/test/resources/alma-fix/990001412590206441.json (hunk 2 above)
Comment (TobiasNx) [reply src/test/resources/alma-fix/990001412590206441.json]: Forgot to run the tests again. Fixed now.

File: src/main/resources/alma/fix/relatedRessourcesAndLinks.fix (hunk 1 above)
Comment (TobiasNx) [reply src/main/resources/alma/fix/relatedRessourcesAndLinks.fix:305]: I agree I just used one of your suggestions from https://github.com/hbz/lobid-resources/issues/1677)

**Comment by TobiasNx** [comment]
wait, I do this PR again. Sorry.

File: src/main/resources/alma/fix/relatedRessourcesAndLinks.fix (hunk 1 above)
Comment (blackwinter) [reply src/main/resources/alma/fix/relatedRessourcesAndLinks.fix:305]: I'm sorry, but this wasn't one of the suggestions. `003=DE-655` was.

Comment (TobiasNx) [reply src/main/resources/alma/fix/relatedRessourcesAndLinks.fix:305]: Thanks for the hint.

**Pull request closed without merging**